scripts/drone/utils/images.star

```diff
@@ -20,21 +20,16 @@ images = {
     "ubuntu": "ubuntu:22.04",
     "curl": "byrnedo/alpine-curl:0.1.8",
     "plugins_slack": "plugins/slack",
-    "python": "python:3.8",
     "postgres_alpine": "postgres:12.3-alpine",
     "mimir": "grafana/mimir-alpine:r316-55f47f8",
     "mysql8": "mysql:8.0.32",
     "redis_alpine": "redis:6.2.11-alpine",
     "memcached_alpine": "memcached:1.6.9-alpine",
     "package_publish": "us.gcr.io/kubernetes-dev/package-publish:latest",
-    "openldap": "osixia/openldap:1.4.0",
```

Comment:
Looks like `openldap` may be needed here `scripts/drone/services/services.star`?

Reply:
hrm yeah actually also some are needed by the enterprise build too... surprised nothing broke there 🤔

Reply:
@baldm0mma it is, but that function isn't referenced anywhere 😂 - I'll remove it too.

Reply:
LOL! Classic. 🤣